src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body,
+.nav__list {
+  margin: 0;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__link {
+  position: relative;
+  height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.logo,
+.nav__link {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

Comment:
Let's combine this logic and use only one selector? Try to figure out how you can do this)

Reply:
done
thanks for the idea)